Tests/Filters/MediaTextEnhancedTest.php

```diff
@@ -0,0 +1,88 @@
+<?php
+namespace Bcgov\Theme\Block\Tests\Filters;
+
+use Bcgov\Theme\Block\Tests\CommonTestCase;
+use Bcgov\Theme\Block\Filters\MediaTextEnhanced;
+
+/**
+ * Test for the MediaTextEnhanced filter
+ */
+class MediaTextEnhancedTest extends CommonTestCase {
+    /**
+     * The MediaTextEnhanced instance.
+     *
+     * @var MediaTextEnhanced
+     */
+    protected $media;
+
+    /**
+     * Used for setup of each test.
+     *
+     * @inheritDoc
+     */
+    public function setUp() :void {
+        parent::setUp();
+        $this->media = new MediaTextEnhanced();
+    }
+
+
+    /**
+     * Test the button with no block contents, should return empty string.
+     *
+     * @return void
+     */
+    public function test_Add_Media_Text_With_Attributes_no_block_content(): void {
+        $this->assertEquals( '', $this->media->add_media_text_attributes() );
+    }
+
+
+    /**
+     * Test to convert button, with no attributes, but update for the image size class.
+     *
+     * @return void
+     */
+    public function test_Add_Media_Text_With_Attributes_with_no_atts(): void {
+        $block            = [
+            'blockName' => 'core/media-text',
+            'attrs'     => [],
+        ];
+        $block_content    = '<img src="https://gov.bc.ca/test.jpg" />';
+        $expected_results = '<img   data-print-width="25" src="https://gov.bc.ca/test.jpg" />';
+        $this->assertEquals( $expected_results, $this->media->add_media_text_attributes( $block_content, $block ) );
+    }
+
+    /**
+     * Test to convert button, with a svg icon to add icon classes.
+     *
+     * @return void
+     */
+    public function test_Add_Media_Text_With_Attributes_as_button_with_title(): void {
+        $block            = [
+            'blockName' => 'core/media-text',
+            'attrs'     => [
+                'title' => 'This is a title',
```

Comment:
Kind of a minor bug with the original function but if you wanted to have a title of 'none' it wouldn't work because it treats 'none' as not having a value.

Reply:
> Kind of a minor bug with the original function but if you wanted to have a title of 'none' it wouldn't work because it treats 'none' as not having a value.

I think because the printWidth for the image and the size for the button are in the defaults it is enough for the test, but thanks for taking the time to review the actually code it was testing.

Reply:
I think without testing printWidth you don't have 100% code coverage of the function, specifically this line: https://github.com/bcgov/bcgov-wordpress-block-theme/blob/development/src/Filters/MediaTextEnhanced.php#L53